core/src/main/web/app/mainapp/mainapp.js

```diff
@@ -70,6 +70,9 @@
           if (!($scope.$$phase || $scope.$root.$$phase))
             $scope.$digest();
         };
+        var getLoadingStatusMessage = function() {
```

Comment:
It's in our styleguide - http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml?showone=Function_Declarations_Within_Blocks#Function_Declarations_Within_Blocks